<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseConfig</class>
 <widget class="QDialog" name="DatabaseConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DatabaseConfig</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MySQL </string>
    </property>
    <property name="icon">
     <iconset resource="../resources/Resources.qrc">
      <normaloff>:/icons/1444503647_mysql-2048-black.png</normaloff>:/icons/1444503647_mysql-2048-black.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>13</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Database Driver:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtServer">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>49</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Servidor / IP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtUsuario">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>79</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Usu√°rio:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtSenha">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>109</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Senha:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>172</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Porta:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbPorta">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50000</number>
   </property>
   <property name="value">
    <number>3306</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="icon">
    <iconset resource="../resources/Resources.qrc">
     <normaloff>:/icons/Ok.png</normaloff>:/icons/Ok.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="icon">
    <iconset resource="../resources/Resources.qrc">
     <normaloff>:/icons/Cancel.png</normaloff>:/icons/Cancel.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtDatabase">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>139</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Banco de Dados:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
